sdk/all/pr.client.yml

```diff
@@ -0,0 +1,23 @@
+# DO NOT EDIT THIS FILE
+# This file is/may be generated automatically and any changes may be lost.
+
+
+pr:
+- master
+
+trigger:
+  branches:
+    include:
+    - users/mitchdenny/pr-pipelines
```

Comment:
Is it necessary to specify the branches?

Reply:
No I'll remove that in a subsequent PR. I was using this for testing purposes as I was debugging the script. I can remove now if you would prefer.

Reply:
I went ahead and removed it because as soon as we merge this PR I'll end up putting in another straight away to remove it - may as well do it now.